recipes/mpi-serial/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "mpi-serial" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MCSclimate/mpi-serial/archive/refs/tags/MPIserial_{{ version }}.tar.gz
+  sha256: cc55e6bf0ae5e1d93aafa31ba91bfc13e896642a511c3101695ea05eccf97988
+  patches:
+    - 0001-Rearrange-code-and-other-minor-fixes-needed-for-comp.patch
+    - fix-makefile.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    # Unix
+    - {{ compiler('c') }}  # [unix]
+    - {{ compiler('cxx') }}  # [unix]
+    - {{ compiler('fortran') }}  # [unix]
+    - coreutils  # [unix]
+    - make  # [unix]
+    # Win
+    - {{ compiler('m2w64_c') }}  # [win]
+    - {{ compiler('m2w64_cxx') }}  # [win]
+    - {{ compiler('m2w64_fortran') }}  # [win]
+    - m2-autoconf  # [win]
+    - m2-coreutils  # [win]
+    # - m2-grep  # [win]
+    - m2-make  # [win]
+    # - m2-sed  # [win]
+    # - m2-which  # [win]
+    # - m2w64-pkg-config  # [win]
+    # - m2w64-dlfcn  # [win]
+  host:
+  run:
+
+# test:
+#   commands:
+#     - pip check
+
+about:
+  home: https://github.com/MCSclimate/mpi-serial
+  summary: MPI library for single processor
+  description: |
+    This library provides a one-processor version of MPI.  Most common MPI calls,
+    including all that are necessary for the Model Coupling Toolkit, are supported.
+    This includes sends and receives (which cannot be simply stubbed out).
+  license: XXX
+  license_family: BSD
```

Comment:
Hm. I think in these cases we should use a `LicenseRef-` expression. I have added `LicenseRef-BSD-ANL`, but @mahf708, I think it would be good to follow up with upstream to clarify how they think about SPDX and how the package fits in. Could you do that?

Reply:
@mahf708 and @rljacob, is this something you can (or have already) discussed?